pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -366,7 +366,7 @@ func (plugin *kubenetNetworkPlugin) setup(namespace string, name string, id kube
 		// initialization
 		shaper := plugin.shaper()
 
-		ingress, egress, err := bandwidth.ExtractPodBandwidthResources(pod.Annotations)
+		ingress, egress, err := bandwidth.ExtractPodBandwidthResources(annotations)
```

Comment:
Should move the bandwidth shaping logic out of the `if plugin.host.SupportsLegacyFeatures` block. Otherwise, this is not run for dockershim.